Tests/Common/Util/LogTest.swift

```diff
@@ -0,0 +1,22 @@
+@testable import Common
+import Foundation
+import SharedTests
+import XCTest
+
+class LogTest: UnitTest {
+    func test_getLogLevel_givenAllStringValues_expectLogLevel() {
+        for logLevelCase in CioLogLevel.allCases {
+            let givenString = logLevelCase.rawValue
+            let expected = logLevelCase
+            let actual = CioLogLevel.getLogLevel(for: givenString)
+            XCTAssertEqual(expected, actual)
+        }
+    }
+
+    func test_loglevel_givenIncorrectString_expectLogLevel() {
+        let givenLogLevelError: CioLogLevel? = nil
+        let expectedLogLevelError = CioLogLevel.getLogLevel(for: "debuggable")
```

Comment:
```suggestion
        let expected: CioLogLevel? = nil
        let actual = CioLogLevel.getLogLevel(for: "debuggable")
```

The order of the 2 variables should be swapped. Our test gives "debuggable" as the test value and we then expect to get back `nil`. 

But, great test besides that.